templates/jaasai/index.html

```diff
@@ -5,7 +5,15 @@
 
 {% block content %}
 
-  <div class="p-strip--suru-bottom has-accent is-dark" style="background-size: auto 100%, cover; background-position: right center; background-image: url('https://assets.ubuntu.com/v1/a9dab044-suru-right.svg'), linear-gradient(266deg, #044f66, #022935)">
+<div class="p-strip is-shallow">
+  <div class="row">
+    <div>
+      <strong>We've released our new dashboard!</strong> To access it, <a href="/models">view your models</a> or for more info, read our <a class="p-link--external" href="#">post on Discourse.</a>
```

Comment:
you should add `p-link--external` on this "view your models" if you have it in the navigation